### DISCUSSION TITLE
add info about how long the feeds are stored in database

### DISCUSSION THREAD

**Discussion by victorhck** [Athou/commafeed#1560] [Ideas]
Once you open an account in commafeed.com you can't find how long the feeds are stored in database or if they are deleted after months or such, or if starred feeds are deleted

Maybe should be added a text with information about this point. Or even  a way to choose whet delete, maybe I don't need feeds older than a month or two.

(just a suggestion)

**Comment by Athou** [comment]
Sure, that makes sense indeed, I'll add that when I get the chance.

**Reply by victorhck** [reply]
Great!! :)

**Reply by Athou** [reply]
Done: https://github.com/Athou/commafeed/discussions/1567

**Reply by I-nnana** [reply]
Hi @Athou , I've been going through your issue and thought there might be a few features missing (hopefully not too dumb):

Wouldn't it be possible to have a button to delete feeds next to the star or share button? And perhaps a limit on the number of feeds, say five hundred instead of a thousand.

I think it would also be nice to be able to set a time limit for the subscription, like 10 days instead of a year. 

(yet another suggestion :) )
